internal/pkg/describe/status.go

```diff
@@ -95,6 +95,23 @@ type ecsServiceStatus struct {
 	Alarms                   []cloudwatch.AlarmStatus `json:"alarms"`
 	StoppedTasks             []awsecs.TaskStatus      `json:"stoppedTasks"`
 	TargetHealthDescriptions []taskTargetHealth       `json:"targetHealthDescriptions"`
+
+	rendererConfigurer rendererConfigurer // The Renderer interface is not mock friendly. We abstract away the code with an intermediate interface so that it can be mocked.
```

Comment:
Why is it not mock friendly? 😂 

Would all this problem go away if we didn't try to mock the Render call? i.e. what happens if in the actual code we do:
```go
bar := summarybar.New(....)
bar.Render(out)
```

Reply:
ooo the reason why I said "not mock-friendly" is because there is no direct way to check if the `data` (of type `Datum`), and `opts` that get passed in is expected. 

The whole point of the weird looking `rendererConfigurer` is to add an intermediate layer that check whether the data is okay.